.env

```diff
@@ -4,3 +4,7 @@ POSTGRES_HOST=resources-postgres
 POSTGRES_DB=resources-psql
 FLASK_ENV=development
 FLASK_DEBUG=1
+SEARCH_USER=search_user
+SEARCH_KEY=search_key
```

Comment:
The [Algolia docs](https://www.algolia.com/doc/api-client/getting-started/instantiate-client-index/#init-index) call this the `Application ID` and `API Key` rather than the Search User and Search Key. Can we rename it such that it matches their terms?